src/components/option-separator/README.md

```diff
@@ -0,0 +1,29 @@
+# OptionSeparator
+
+OptionSeparators are used with the Multiselect and Option components. They provide you with a separation between Options.
+
+## Props
+
+| Prop | Type | Description |
+| ---- | ---- | ----------- |
+| children | string | Custom string, which is displayed to the user, functions as a separator between set of groups of Options. |
```

Comment:
I don't really get this sentence.

`children is the content which takes place within the separator component.`